lib/src/backend/test_platform.dart

```diff
@@ -109,6 +140,14 @@ class TestPlatform {
   Object serialize() {
     if (builtIn.contains(this)) return identifier;
 
+    if (parent != null) {
```

Comment:
[optional] create the map with the fields name and identifier then conditionally append the parent field before returning.

Reply:
I think that would make it less clear that the child map isn't the same structure as the base-level map.